DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Indigo\Bundle\CookieConsentBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * @author Márk Sági-Kazár <mark.sagikazar@gmail.com>
+ */
+class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('indigo_cookie_consent');
+
+        $rootNode
+            ->children()
+                ->arrayNode('options')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('message')->defaultValue('message')->end()
+                        ->scalarNode('dismiss')->defaultValue('dismiss')->end()
+                        ->scalarNode('learnMore')->defaultValue('learn_more')->end()
```

Comment:
If you disable translation, you can pass any text here which is then displayed. However this is one of the things I am not sure about: we have translation, should we allow to override labels in the config? Or force users to override translation domains instead?